x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/process/normalizer/Normalizer.java

```diff
@@ -198,15 +198,15 @@ private double mergeRecursively(Iterator<NormalizerResult> scoresIter, Normaliza
      * @return true if the update is considered "big"
      */
     private static boolean isBigUpdate(double oldVal, double newVal) {
-        if (Math.abs(oldVal - newVal) >= 1.0) {
+        if (Math.abs(oldVal - newVal) >= 2.0) {
```

Comment:
I think we should take the chance that we experimented with those numbers and add some commentary that explains why they were picked. It will be very helpful next time we have to do a similar recalibration.

Reply:
Thanks @dimitris-athanasiou. Yes agreed there is still some room for experimentation here. With this choice the number of updates written to the ES index is effectively halved producing a significant performance improvement. I'll add some comments explaining this in the code as you suggest.